lib/chef/knife/kvm_base.rb

```diff
@@ -33,6 +33,7 @@ def run_remote_command(command)
               if success
                 channel.on_data do |ch, data|
                   return_val = data.chomp
+                  print return_val if console
```

Comment:
Are we live streaming the output here? 

I am not fully clear what `console` means. Also where is the `print()` call coming from. Looks like we are using `ui.*`  methods in other places. Should we use the appropriate one instead of `print()` here?